main.tex

```diff
@@ -180,6 +181,117 @@ \section*{Introduction}
+\subsection*{Data collection and processing}
+
+
+%
+For detailed information on Ag1000g WGS sample collection, sequencing, variant calling, quality control and phasing see Ag1000g Consortium \emph{et al}. (\cite{Ag1000gConsortium2017}).
+%
+In brief, \emph{An. gambiae} and \emph{An. coluzzii} mosquitoes were collected from eight countries across Sub-Saharan Africa: Angola, Burkina Faso, Cameroon, Gabon, Guinea, Guinea Bissau, Kenya and Uganda. 
+%
+Guinea Bissau was an \emph{An. gambiae}/\emph{coluzzii} admixed population, from Angola just \emph{An. coluzzii} were sampled, Burkina Faso had samples of both \emph{An. gambiae} and \emph{An. coluzzii} and all other populations consisted of purely \emph{An. gambiae}.
+%
+Mosquitoes were individually whole genome sequenced on the Illumina HiSeq 2000 platform, generating 100bp paired-end reads. 
+%
+Sequenced reads were aligned to the A. gambiae AgamP3 reference genome assembly \cite{Holt2002}). 
+%
+Aligned bam files underwent improvement, before variants were called using GATK UnifiedGenotyper. 
+%
+Quality control included removal of samples with mean coverage <= 14x and an accessibility map was employed following a similar approach to that used for human data by The 1000 Genomes Project Consortium \cite{The1000GenomesProjectConsortium2010}). 
+%
+Variant filtering rules removed accessible variants if: FS < 60.0, HRun > 4, DP <18000 or >32000, MQ < 40, alleles > 2, QD < 5, ReadPosRankSum < -8, overlapped positions repeat masked by DUST (http://blast.wustl.edu/pub/dust). 
+%
+This process produced a call set containing WGS sequences for 765 wild caught individuals which was publicly released \cite{Ag1000gConsortium2017}. 
+%%
+
+%
+The Ag1000g variant data was functionally annotated using the SnpEff v4.1b software which allowed investigation of potential phenotype altering variants within Vgsc \cite{Cingolani2012}.
+%
+Non-synonymous \emph{Vgsc} variants were identified as all variants in AGAP004707, 2L:2358158-2431617, with a SnpEff annotation of “missense”  and an ALT allele frequency of >5\% in at least one of the nine mosquito populations, with the exceptions of the multi-allelic SNP 2L:2400071 G>A which is shown despite only being found in \emph{An. gambiae} from Cameroon at 0.4\% frequency, as the G>T variant at the same position which causes the same codon change (\texttt{M490I}), is found above 5\% frequency in Kenya.
+%
+\texttt{F1920S} is included for continuity with recent \emph{An. gambiae Vgsc} research \cite{Ag1000gConsortium2017}.
+%
+A minimum ALT allele frequency was employed to discriminate towards variants that may be undergoing selective sweeps and against less informative low frequency alleles.
+%%
+
+%
+For ease of comparison with previous work on \emph{Vgsc}, pan Insecta, \emph{An. gambiae} codon numbering is also reported in the equivalent \emph{Musca domestica} codon numbering, the species in which the gene was first discovered.
```

Comment:
Maybe also reference Table 1 here, e.g., "For ease of comparison with previous work on \emph{Vgsc}, pan Insecta, in Table \ref{table:variants_missense} we report codon numbering for both \emph{An. gambiae} and \emph{Musca domestica} (the species in which the gene was first discovered)."